philadelphia/src/test/java/org/jvirtanen/philadelphia/FIXSessionTest.java

```diff
@@ -0,0 +1,20 @@
+package org.jvirtanen.philadelphia;
+
+import org.junit.Test;
+
+/**
+ * Created on: 03/24/2016
+ *
+ * @author Alexander Aleshin
+ * @version %I%, %G%
+ */
+public class FIXSessionTest {
+    
+    @Test
+    public void beginStringLength() {
+        FIXConfig FIXT11Config = new FIXConfig.Builder()
```

Comment:
Variable names usually start with a lowercase letter. Perhaps `config` would be sufficient here?